rollup.config.js

```diff
@@ -49,25 +55,28 @@ const plugins = [
     preventAssignment: true,
     values: {
       'process.env.NODE_ENV': JSON.stringify(dev ? 'development' : 'production'),
-    }
+    },
   }),
   watch && serve(serveopts),
   !dev && terser(),
 ];
 
-export default [
```

Comment:
I'm removing the array to make it match the examples in rollup website. Both forms are accepted though.